config/locales/en.yml

```diff
@@ -154,6 +154,7 @@ en:
       header_message: 'This page cannot be displayed'
     not_found:
       header_message: 'This page does not exist'
+    empty_params: 'Can''t be blank'
```

Comment:
So you'll need to translate this to the other locales! You can use Google translate for now!